app/src/main/res/color/touch_text_color.xml

```diff
@@ -0,0 +1,5 @@
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Generally selectors are not saved in `color/` folder, they go to a folder that is called `drawable`. Please put it there